modules/apps/account/account-admin-web/src/main/resources/content/Language.properties

```diff
@@ -53,8 +53,8 @@ there-are-no-users-associated-with-this-account=There are no users associated wi
 there-are-no-users-associated-with-this-role=There are no users associated with this role.
 this-account-does-not-have-a-parent-account=This account does not have a parent account.
 this-account-does-not-have-a-valid-domain=This account does not have a valid domain.
-this-email-has-invalid-domain=This email has invalid domain.
 this-user-does-not-belong-to-any-accounts=This user does not belong to any accounts.
 user-display-data=User Display Data
 valid-domains=Valid Domains
-view-valid-domains=View valid domains
\ No newline at end of file
+view-valid-domains=View valid domains
+x-is-not-a-valid-domain-for-the-following-accounts-x={0} is not a valid domain for the following accounts: {1}.
```

Comment:
@dejuknow this wording still feels odd to me. It makes me feel like I'm supposed to provide a domain that's valid in all accounts, and I "failed in the following cases". I'm not even sure we need to list the account names, since we do let them see which domains are valid.